setup.py

```diff
@@ -47,7 +47,7 @@
     python_requires='>=2.7',
     install_requires=[
         "pycrunch==0.5.5",
-        "requests==2.27.0",
```

Comment:
Can you use the `if PY27` to use the pinned version and use unpinned otherwise? I am a tad concerned that this may make python2.7 installations that YG need problematic.

Reply:
Great Idea @jjdelc! 😃

I've updated as per your feedback.